src/test/java/BoardTest.java

```diff
@@ -6,11 +6,23 @@
 import static org.junit.jupiter.api.Assertions.*;
 
 class BoardTest {
+
     @Test
     void returnsGrid() {
         var grid = new Grid
-                (new ArrayList<String>(Arrays.asList("1", "2", "3", "4", "5", "6", "7", "8", "9")));
+                (new ArrayList<>(Arrays.asList
+                        ("1", "2", "3", "4", "5", "6", "7", "8", "9")));
```

Comment:
This feels a bit clunky. Might be a nice opportunity for a static method like:
`Grid.from("1", "2", "3", "4", "5", "6", "7", "8", "9")` (see https://www.geeksforgeeks.org/variable-arguments-varargs-in-java/)

or at least update the Grid constructor to take a `List<String>`, which could then be:
```
public Grid(List<String> squares) {
  this.squares = new ArrayList<>(squares);
```

Reply:
Thanks, Matt. I'll look into this